devel/fetch_metrics/fetch_metrics.py

```diff
@@ -239,14 +264,25 @@ def main():
 
     # Define job cache and (parallel) job handler
     jc = hi.JobCache(feed_name='default-job-cache')
-    jh = hi.ParallelJobHandler(num_workers=args['workercount'])
 
-    with hi.Config(job_cache=jc, job_handler=jh):
-        for sorting_record in sortings:
-            print_per_verbose(2, f"Creating job-pair {count + 1} ({extract_sorting_reference_name(sorting_record)})")
-            process_sorting_record(sorting_record, comparison_list)
-            count += 1
-            if args['test'] > 0 and count >= args['test']: break
+    # Set up the job handler
+    # example srun_command: srun --exclusive -n 1 -p <partition>
+    srun_command = os.getenv('HITHER_SRUN_COMMAND', None)
```

Comment:
srun command comes from the HITHER_SRUN_COMMAND environment variable. If this is set, it uses slurmjobhandlers. Else uses paralleljobhandler.